libparsec/crates/client/src/client/mod.rs

```diff
@@ -533,8 +541,175 @@ impl Client {
     pub async fn accept_tos(&self, tos_updated_on: DateTime) -> Result<(), ClientAcceptTosError> {
         tos::accept_tos(self, tos_updated_on).await
     }
+
+    pub async fn export_recovery_device(
```

Comment:
You should put your code in a dedicated `client/recovery_device.rs` module in order to keep this file readable considering how many method `Client` has